app/modules/import-export/processes/import.php

```diff
@@ -177,6 +179,10 @@ public function __construct( string $path, array $settings = [], array $old_inst
 
 			$this->set_default_settings();
 		}
+
+		add_filter( 'wp_php_error_args', function ( $args, $error ) {
+			return $this->filter_php_error_args( $args, $error );
```

Comment:
Why wrap in anonymous and not make function public and do `( 'wp_php_error_args', [ $this, 'filter_php_error_args' ] )` ?

Reply:
I'm trying to avoid making functions public, no one should change it.